test/components/DayPicker_spec.jsx

```diff
@@ -832,6 +833,17 @@ describe('DayPicker', () => {
     });
   });
 
+  describe('#getWeekHeaders', () => {
+    it('returns unmuted weekday headers for currentMonth in a future', () => {
+      const INITIAL_MONTH = moment().add(2, 'Months').week(3).weekday(3);
+      const dayPicker = PureDayPicker.prototype.constructor(PureDayPicker.defaultProps);
+      dayPicker.state.currentMonth = moment(INITIAL_MONTH);
```

Comment:
it'd be better to use an enzyme wrapper here, rather than manually constructing an instance.

Reply:
I wanted to prevent render to call `getWeekHeaders()`, because it was changing state already on that stage.

Reply:
`sinon.spy(PureDayPicker.prototype, 'render', () => null)`?